### ISSUE TITLE
AWS Two steps login with OTP code

### ISSUE THREAD

**Issue by Hideman85** [buttercup/buttercup-browser-extension#379]
AWS login page is in two steps it first ask for the username & password and then if your MFA is enabled it ask to input the OTP.

![image](https://user-images.githubusercontent.com/16165119/98090985-af414380-1e7c-11eb-9bc8-40e3cec08300.png)

When you try to use the very nice auto login feature it fail since we needs the two steps to be done.

Related issue:
 - https://github.com/buttercup/buttercup-browser-extension/issues/374
 - https://github.com/buttercup/buttercup-browser-extension/issues/268

**Comment by perry-mitchell** [comment]
Yes, this is part of a larger issue: Multi-step logins. They're super ugly in my opinion, and quite useless. Would be easiest if they were all on the same "page". Regardless, we need to:

 * Recognise more fields (MFA, account IDs etc.)
 * Detect multi-stage logins and fill those too

**Comment by perry-mitchell** [comment]
OTP logins supported in V3. Not multi-stage logins yet, however.
